doc/make.jl

```diff
@@ -299,9 +362,12 @@ else
         collapselevel = 1,
         sidebar_sitename = false,
         ansicolor = true,
+        size_threshold = 800 * 2^10, # 800 KiB
+        size_threshold_warn = 200 * 2^10, # the manual has quite a few large pages, so we warn at 200+ KiB only
     )
 end
 
+@info read(`$(Sys.which("git")) --version`, String)
```

Comment:
I see you added a dep on `LibGit2` for `LibGit2.GITHUB_REGEX` here and Documenter.jl also [depends on LibGit2](https://github.com/JuliaDocs/Documenter.jl/blob/ae0188b2df02bc0e3f51f649832017abfccea3b3/Project.toml#L15). Is this info statement because Documentor also uses git via shelling out? If not it may be misleading. (somewhat related: https://github.com/JuliaLang/Pkg.jl/issues/2679)

Reply:
This is just a debug print and I'll remove this. Generally, I need to do a pass on this PR before we merge, to clean it all up.

The issue I ran into here was that Git on the Windows runners is ancient (and I was trying to figure out what version it is exactly). But I resolved that by switching Documenter to Git.jl (which is the better than relying on the system's `git` binary anyway).

Reply:
Just a note here though: kinda silly to depend on LibGit2 just for this regex nowadays. https://github.com/JuliaDocs/Documenter.jl/issues/2363